website/static/js/filepage/revisions.js

```diff
@@ -44,6 +45,7 @@ var FileRevisionsTable = {
                 url: self.file.urls.revisions,
                 beforeSend: $osf.setXHRAuthorization
             }).done(function(response) {
+                response = fangorn.DefaultOptions.lazyLoadPreprocess.call(this, response);
```

Comment:
@felliott As near as I can tell there isn't a treebeard object available in revisions.js , am I missing something?